webtau-feature-testing/examples/scenarios/concept/runOnlySelected.groovy

```diff
@@ -12,8 +12,12 @@ sscenario('step two') {
     // test that you want to focus on
 }
 
-scenario('step three') {
+singleScenario('step three') {
```

Comment:
the reason for `sscenario` name was not really because it was `single`. I "borrowed" it from Jasmine, where `it` was a test declaration, and `iit` was run only this.
I like `singleScenario` but not sure how it translates when there are multiple `single` scenarios?

Reply:
Yeah, I wondered that too but struggled to come up with a better name.  I suppose technically it does not need to start with an `s` but I think it would be preferable.

Reply:
Looks like Jasmine has moved away from that pattern to call these "focused" specs: https://jasmine.github.io/2.1/focused_specs.html.  Not sure I like that.